### PR TITLE
Add Open Achievements Hotkey

### PR DESCRIPTION
**Pull request by LillyJadeKatrin**
Adds a hotkey to pause emulation and bring up the Achievements dialog.

### CODE REVIEW

File: Source/Core/DolphinQt/MainWindow.cpp [hunk 1]
```diff
@@ -649,6 +649,15 @@ void MainWindow::ConnectHotkeys()
     movie.SetReadOnly(read_only);
     emit ReadOnlyModeChanged(read_only);
   });
+#ifdef USE_RETRO_ACHIEVEMENTS
+  connect(
+      m_hotkey_scheduler, &HotkeyScheduler::OpenAchievements, this,
+      [this] {
+        Pause();
```
Comment (JosJuice) [inline Source/Core/DolphinQt/MainWindow.cpp:656]: Is there any particular reason why this pauses emulation? I'm not aware of any other hotkeys that work like this.

**Review by JosJuice — APPROVED** [review]

**Comment by AdmiralCurtiss** [comment]
Please squash the `HotkeyScheduler.cpp` changes from the second commit into the first.

**Comment by JosJuice** [comment]
Oh, yes, I missed that.

File: Source/Core/Core/HotkeyManager.cpp [hunk 2]
```diff
@@ -37,7 +37,10 @@ constexpr std::array<const char*, NUM_HOTKEYS> s_hotkey_labels{{
     _trans("Unlock Cursor"),
     _trans("Center Mouse"),
     _trans("Activate NetPlay Chat"),
-    _trans("Control NetPlay Golf Mode"),
+    _trans("Control NetPlay Golf Mode"),  
+#ifdef USE_RETRO_ACHIEVEMENTS
+    _trans("Open Achievements"),
+    #endif  // USE_RETRO_ACHIEVEMENTS
```
Comment (AdmiralCurtiss) [inline Source/Core/Core/HotkeyManager.cpp:43]: Something... weird is going on with the formatting here. Why is the first line marked as different, and why is the indent on the `#endif` wrong? Lint isn't complaining about this either which confuses me.

Comment (LillyJadeKatrin) [reply Source/Core/Core/HotkeyManager.cpp]: The first line had hidden space at the end of the line. Why the linter never picked up on it is beyond me.

**Pull request merged**